javalin/src/main/java/io/javalin/http/HttpResponseException.kt

```diff
@@ -6,66 +6,68 @@
 
 package io.javalin.http
 
-open class HttpResponseException @JvmOverloads constructor(val status: Int, message: String, val details: Map<String, String> = mapOf()) : RuntimeException(message)
+import io.javalin.http.HttpCode.*
+
+open class HttpResponseException @JvmOverloads constructor(val status: HttpCode, message: String = status.message, val details: Map<String, String> = mapOf()) : RuntimeException(message)
```

Comment:
1. I'd call `status` -> `code`, so it won't look like `e.status.status` if someone would like to get its number
2. Might be a good idea to put each parameter in new line for better readability